azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkit/lib/appservice/AzureAppService.java

```diff
@@ -109,7 +109,8 @@ public IWebApp webapp(WebAppEntity webAppEntity) {
     }
 
     public @Nonnull List<Runtime> listWebAppRuntimes(@Nonnull OperatingSystem os, @Nonnull JavaVersion version) {
-        return Runtime.values().stream().filter(r -> Objects.equals(os, r.getOperatingSystem()) && Objects.equals(version, r.getJavaVersion()))
+        return Runtime.values().stream()
```

Comment:
please filter out function runtime here, we may need some static method to get runtime for services in `Runtime`